packages/neuron-wallet/src/utils/createLoopTask.ts

```diff
@@ -0,0 +1,37 @@
+import { BrowserWindow } from 'electron'
+import env from '../env'
+import SyncBlocksService from '../services/syncBlocks'
+import initConnection from '../typeorm'
+
+// create a background task to sync transactions
+// this task is a renderer process
+const createLoopTask = () => {
+  let loopWindow: BrowserWindow | null = new BrowserWindow({
+    x: 10,
+    y: 10,
+    show: false,
+    webPreferences: {
+      nodeIntegration: false,
```

Comment:
According to the doc https://electronjs.org/docs/tutorial/multithreading#multi-threaded-nodejs, the `nodeIntegrationInWorker` should be set to `true`

Reply:
It seems can set to true, but not must set to true, and why we should set to true

Reply:
> It seems can set to true, but not must set to true, and why we should set to true

I'm not sure, doesn't the native module requires node runtime?

Reply:
It works well, we can add this if we need in the future.